cfpb/consumerfinance.gov
# Remove static legacy student pages

## Removals

- Deletes legacy template for https://www.consumerfinance.gov/practitioner-resources/students/knowbeforeyouowe/ so it can be delivered via Wagtail.
- Deletes legacy template for https://www.consumerfinance.gov/practitioner-resources/students/helping-borrowers-find-ways-to-stay-afloat/ so it can be delivered via Wagtail.

## Changes

- ~~Adds redirect for `/knowbeforeyouowe/` to `/helping-borrowers-find-ways-to-stay-afloat/`.~~


## How to test this PR

1. Refresh database and publish http://127.0.0.1:8000/admin/pages/15044/edit/ and http://127.0.0.1:8000/admin/pages/15045/edit/
2. Wagtail pages should exist at  https://127.0.0.1:8000/practitioner-resources/students/knowbeforeyouowe/ (redirected) and https://127.0.0.1:8000/practitioner-resources/students/helping-borrowers-find-ways-to-stay-afloat/
